src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -175,10 +217,13 @@ public PersonListPanel getPersonListPanel() {
     private CommandResult executeCommand(String commandText) throws CommandException, ParseException {
         try {
             CommandResult commandResult = logic.execute(commandText);
-            logger.info("Result: " + commandResult.getFeedbackToUser()); // used to log the result, not actually
-            // used to display
+            logger.info("Result: " + commandResult.getFeedbackToUser());
             resultDisplay.setFeedbackToUser(commandResult.getFeedbackToUser());
 
+            if (commandText.trim().toLowerCase().startsWith("view")) {
```

Comment:
Might be worth refactoring the predicate condition to a `isViewCommand()` method, similar to the `isShowHelp()` method.